### ISSUE TITLE
Test login capabilities from ModelCatalogIngestion API

### ISSUE THREAD

**Issue by dhruvp-8** [mintproject/MINT-ModelCatalogExplorer#90]

**Comment by sirspock** [comment]
You can generate the client using openapi-generator. These are the [supported languagues](https://github.com/OpenAPITools/openapi-generator/tree/master/docs/generators)
and this is an example using the javascript generator 
[sirspock/MINT-ModelCatalogIngestionAPI-client-javascript](https://github.com/sirspock/MINT-ModelCatalogIngestionAPI-client-javascript)

**Comment by maurya-rohit** [comment]
@sirspock I am getting CORS policy error which is : "Access to XMLHttpRequest at 'https://api.models.mint.isi.edu/v0.0.2/user/login?username=def&password=dg&_=1557185659037' from origin 'http://127.0.0.1:8081' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource." Is there anything needs to be done in the header. I tried different thing but getting the same error.


Here is my ajax request: 
{ crossOrigin:true,
            url: "https://api.models.mint.isi.edu/v0.0.2/user/login",
            type: "GET",
            data: {username: some.Username,password:some.Password},
            cache: false,
            timeout: 5000,
            async: true,
            headers:{
                "Access-Control-Allow-Origin":"*",
                "Access-Control-Allow-Methods": "POST, GET, OPTIONS, PUT, DELETE, HEAD",
            }

**Comment by sirspock** [comment]
Fixed

```sh
➜  ~ curl -I -X GET https://api.models.mint.isi.edu/v0.0.2/user/login
HTTP/2 400
server: nginx/1.15.12
date: Wed, 08 May 2019 23:47:00 GMT
content-type: application/problem+json
content-length: 121
access-control-allow-origin: *
```

**Comment by dgarijo** [comment]
@sirspock, please don't close the issue until @maurya-rohit or @dhruvp-8 have implemented the login :)

**Comment by sirspock** [comment]
Sorry, I thought it was another issue.
